pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/MaxHitRateTracker.java

```diff
@@ -80,4 +82,14 @@ int getMaxCountPerBucket(long now) {
     _lastAccessTimestamp = now;
     return maxCount;
   }
+
+  @VisibleForTesting
+  @Override
+  int getHitCount(long now) {
```

Comment:
Should this be `getHitCount()`? Otherwise `finalQueryQuotaEntity.getMaxQpsTracker().getHitCount();` would be calling into which? Am I missing smth here?

Reply:
Good question! The call of `finalQueryQuotaEntity.getMaxQpsTracker().getHitCount()` will call the `getHitCount()` method from the base class, which internally invokes the `int getHitCount(long now)` method in the child class. So we should be good here.

Reply:
ohh I see make sense